card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
+  padding: 40px;
+  gap: 20px;
+  /* border-radius: 15px; */
+}
+
+.content-section {
```

Comment:
Add component name prefix (card-content-section)

Reply:
**Resolved:** Added prefix to all component and committed